src/google-classroom-extractor/edfi_google_classroom_extractor/mapping/assignment_submissions.py

```diff
@@ -3,10 +3,13 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
-from typing import Dict, Tuple, Any
+from typing import Dict, List, Tuple, Any
+import ast
```

Comment:
What is `ast`?

Reply:
The library: https://docs.python.org/3/library/ast.html
The function: https://docs.python.org/3/library/ast.html#ast.literal_eval

Reply:
I was looking on pypi and saw something really strange. This makes more sense now.